### ISSUE TITLE
Curate rest of ASGS ABS structures

### ISSUE THREAD

**Issue by jyucsiro** [CSIRO-enviro-informatics/asgs-dataset#12]
asgs-dataset implements the main ABS structures (MB, SA1-4, STE). 

This issue to for curating the rest of the ABS Structures as part of the `asgs-dataset`.

![asgs-abs-structures](https://user-images.githubusercontent.com/4723726/70797852-7d089c80-1df9-11ea-987b-0ff4356e2419.jpg)

These are the RA, SOS, SOSR, UCL, IREG, IARE, ILOC, GCCSA, SUA features.

**Comment by dr-shorthair** [comment]
These are all in the ASGS Ontology and also now have the appropriate containment relationships axiomatized - see https://github.com/AGLDWG/asgs-ont/blob/master/images/Figure-1-class-containments.png and https://github.com/AGLDWG/asgs-ont/blob/master/asgs.ttl

**Comment by ashleysommer** [comment]
I have a question about features:
* DestinationZone (DZN),
* StateSuburb (SSC),
* NaturalResourceManagementRegion (NRMR)

In the ASGS WFS endpoint, when querying details of a Meshblock, the data shows the meshblock belonging within a `dzn`, an `ssc`, and a `nrmr`. These class types are present in the asgs ontology, but are not in the diagram above, and are not in [this diagram](https://github.com/AGLDWG/asgs-ont/blob/master/images/Figure-1-class-containments.png).

Update:
I found `ssc` and `nrmr` in [this non-asgs structures diagram](https://www.abs.gov.au/websitedbs/D3310114.nsf/4a256353001af3ed4b2562bb00121564/c453c497aadde71cca2576d300026a38/Body/2.1E7E!OpenElement&FieldElemFormat=jpg):

![this non-asgs structures diagram](https://www.abs.gov.au/websitedbs/D3310114.nsf/4a256353001af3ed4b2562bb00121564/c453c497aadde71cca2576d300026a38/Body/2.1E7E!OpenElement&FieldElemFormat=jpg)

**Comment by dr-shorthair** [comment]
Right. I didn't copy the patterns over to the non-ABS structures yet. What's a 'Destination zone'?? It is not mentioned in the [ASGS reference @ ABS](https://www.abs.gov.au/websitedbs/D3310114.nsf/home/Australian+Statistical+Geography+Standard+(ASGS))

**Comment by ashleysommer** [comment]
DZN definition is in the asgs.ttl file: https://github.com/AGLDWG/asgs-ont/blob/df2dfdddf0a0be9ff9cc0fb5a6a59dc449e37263/asgs.ttl#L200

**Comment by ashleysommer** [comment]
Added new issue for Non-ABS feature types: https://github.com/CSIRO-enviro-informatics/asgs-dataset/issues/18

**Comment by dr-shorthair** [comment]
Ontology supports all these relationships now
http://linked.data.gov.au/def/asgs#DestinationZone
http://linked.data.gov.au/def/asgs#NaturalResourceManagementRegion
http://linked.data.gov.au/def/asgs#StateSuburb
